sp800-63a/sec4_ial.md

```diff
@@ -109,7 +109,7 @@ At a minimum, the applicant must be verified by a process that is able to achiev
 
 #### 4.4.1.5. Presence Requirements
 
-In-person and remote identity proofing are allowed. The CSP SHOULD offer both in-person and remote proofing.
+In-person and unsupervised remote identity proofing are allowed. The CSP SHOULD offer both in-person and unsupervised remote proofing.
```

Comment:
This could be interpreted that supervised remote identity proofing isn't allowed. Suggest leaving these two instances of _unsupervised_ out.